src/api/dependency-discovery/src/github.js

```diff
@@ -0,0 +1,66 @@
+const { request } = require('@octokit/request');
```

Comment:
Can you file a follow-up to use more of https://github.com/octokit/octokit.js vs. just `request`?  The full https://github.com/octokit/octokit.js does a lot more, including throttling, retries, etc.

Reply:
Filed #3324